keycloak/keycloak-admin-ui
# Events search fixes

## Motivation
- Fixed issue #1055
- Fixed issue #1057
- Removed redundant Divider
- Added refresh button
- Added admin search form

## Brief Description
Improved user events search.

## Verification Steps
1. Go to /events user tab and verify increased form width so that the longest event type is fully visible.
2. Select userId or event type and do a search, click the refresh button, and verify that all events display.
3. Select non-existing userId and do a search, verify that grey line display underneath the chips.
4. Select admin events tab and verify that search form exists.

## Checklist:


## Additional Notes
None
